### ISSUE TITLE
information-page

### ISSUE THREAD

**Issue by XxKikoxX** [XxKikoxX/capstone-project#1]
## Information Page

As a User 

I want to see all relevant information. 

so that i know everything about my clothes.
​

## Description

![Image](https://user-images.githubusercontent.com/123958222/227887610-c42a3b02-c3ed-4c04-ad09-54deb2e6743b.png)

​

## Acceptance Criteria

- [x] the information for "kleidung hinterlegt + Uhrzeit" and "location schließt um + Uhrzeit" is displayed
- [x] the information for "id name + id number (check-in id)" is displayed
      ​

## Tasks

- [x] create the component with data for ("Kleidung hinterlegt" + "Uhrzeit")
- [x] create the component with data for ("Benutzername" + "Name der location")
- [x]  writing tests

**Comment by PReichetanz** [comment]
Hi Kevin,

a really good start for your user story! 🎉

💬 Let's discuss some issues which came to my mind while reading:

## Value Proposition

- Please delete the `...` .
- I would rephrase to be more concise:
```
As a user
I want to see all relevant information
so that I know everything about my clothes.
```

## Acceptance Criteria
- There should be no technical wording here ("card component" etc. belongs to the Task section):
  - "Information about 'Kleidung hinterlegt' etc. is displayed."
  - "The time is dynamically changed on each re-render."
  - etc.

## Tasks
- If you want to add tests, write it down here.
